<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>Poster.utf8</title>






<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Special Elite' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 841mm 1189mm;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 841mm;
height: 1189mm;
text-align: justify;
font-size: 45px;
line-height: 1.05;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #00b4d824;
color: #05668d;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #00b4d820;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 13.7px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 841mm;
height: calc(1189mm *  (1 - 0.090 - 0.084 - 0.01) );
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Rasa;
margin-top: calc(1189mm *  0.090 );
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #00b4d8;
  font-weight: bold;
}
a {
text-decoration: none;
color: #00b4d8;
}
#title {
font-size: 125pt;
text-align: left;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
background: 0;
}
#author {
color: #05668d;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-top: 0.1em;
color: ;
font-style: italic;
font-size: 20px;
margin: 0;
}
sup {
color: #02c39a;
}
.affiliation sup {
font-size: 20px;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #00b4d8;
margin: 0;
line-height: 85%;
font-size: 30px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin: 0;
font-weight: bold;
}
.section h1 {
  text-align:center;
  padding-bottom:5px;
  background:
    linear-gradient(
      to left,
      #ffffff 1%,
      #ffffff 20%,
      #05668d75 33%,
      #05668d 50%,
      #05668d75 66%,
      #ffffff 80%,
      #ffffff 99%
    )
    left
    bottom
    #ffffff
    no-repeat;
  background-size:100% 5px ;
  margin-top: 0.5em;
  margin-bottom: 0.5em;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.main {
width: 841mm;
height: calc(1189mm * 0.090);
position: absolute;
background-color: #05668d;
color: #ffffff90;
font-family: Special Elite;
background-image: linear-gradient(#05668d 50%,#00b4d8);
}
.main strong {
color: #ffffff;
}
.main strong > sup {
color: #ffffff;
}
.main sup {
color: #ffffff90;
}
#main-img-left {
width: 10%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
#main-img-center {
width: 10%;
left: calc(841mm * 0.45);
bottom: 0.5in;
position: absolute;
}
#main-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
.main p {
font-size: 150px;
font-family: Special Elite;
text-align: center;
margin: 0;
position: absolute;
top: 50%;
-ms-transform: translateY(-50%);
transform: translateY(-50%);
margin-left: 1em;
}
.fab {
color: #00000030;
font-size: 25px;
}
.twitter, i {
color: #00000030;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000030;
font-size: 35px;
}
.envelope {
color: #00000030;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 841mm;
height: 1189mm;
padding: 0cm;
}
.main_bottom {
width: 841mm;
height: calc(1189mm * 0.084);
margin-top: calc(1189mm * (1 - 0.084));
position: absolute;
background-color: #05668d;
background-image: linear-gradient(#00b4d8 10%, #05668d);
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
span > #tab:mytable {
  font-weight: bold;
}
.orcid img {
  width: 3%;
}
.emphasis {
  background-color: #008080;
  color: #ffffff;
  border: solid #0b2045 3mm;
  margin: 1em;
  padding-left: 0;
  padding-right: 0;
}
.emphasis h1 {
  font-weight: bold;
  background: none;
  background-color: #0b2045;
  padding-bottom: 5mm;
  padding-top: 1mm;
  margin-top: -1mm;
  margin-right: -1mm;
  margin-left: -1mm;
}
.emphasis blockquote {
  border: 0;
}
.emphasis ol {
  padding: 0;
  padding-left: 8%;
  font-size: 100%;
  font-weight: bold;
}
.emphasis p {
  color: #ffffff;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<div class="section">

<h3 id="author" class="author">

Gihawi, A.<sup> 1, <a class="orcid" href="https://orcid.org/0000-0002-3676-5561"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>
<a class="twitter" href="https://mobile.twitter.com/AbrahamGihawi"><i class="fab fa-twitter"></i>&nbsp;&nbsp;@AbrahamGihawi</a><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:A.Gihawi@uea.ac.uk" class="email">A.Gihawi@uea.ac.uk</a> <br>
     </h3>

<h5 id="author_extra", class="author_extra">
 Hurst, R.<sup>1</sup>
 Leggett, R.M.<sup>2</sup>
 Cooper, C.S.<sup>1</sup>
 Brewer, D.S.<sup>1,2</sup>
 Genomics England Research Consortium<sup>3</sup>
</h5>


<p id="affiliation" class="affiliation">
<sup>1</sup> Bob Champion Research and Education Building, University of East Anglia, Norwich, UK<br> <sup>2</sup> Earlham Institute, Norwich, UK<br> <sup>3</sup> Genomics England, London, UK
</p>
</div>

<div id="background" class="section level1">
<h1>Background</h1>
<p>The role of <em>Helicobacter pylori</em><span class="citation">[<a href="#ref-RN102">1</a>]</span> and Human papillomavirus<span class="citation">[<a href="#ref-RN138">2</a>]</span> in gastric and cervical cancer are testament to the prominent role that pathogens can play in cancer. When submitting tumours to whole genome sequencing, it is possible to indicentally sequence microbes in close proximity<span class="citation">[<a href="#ref-RN455">3</a>]</span>. We have been using the 100,000 Genomes Project as a rich resource to search for evidence of microbial DNA.</p>
<p>We benchmarked software to devise the best approach for cancer whole genome sequence metagenomics. The top performing approaches are provided in a tool called <a href="https://github.com/UEA-Cancer-Genetics-Lab/sepath_tool_UEA">SEPATH</a> <span class="citation">[<a href="#ref-RN454">4</a>]</span> which performs the following:</p>
<ul>
<li>Extracts unmapped reads from BAM files</li>
<li>Quality trimming &amp; human read depletion</li>
<li>Metagenomic classification with Kraken<span class="citation">[<a href="#ref-RN72">5</a>]</span></li>
</ul>
<p>Additionally, we have also been investigating the taxonomy and functional potential of contigs produced by metagenomic assembly.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<p>Non-human reads were extracted and classified using <a href="https://github.com/UEA-Cancer-Genetics-Lab/sepath_tool_UEA">SEPATH</a>. Classifications from PCR-free, fresh-frozen samples (<em>N</em>=7,775) with <span class="math inline">\(&lt;20\)</span> reads were filtered. Taxa were removed according to published ‘<em>black lists</em>’ of common contaminants<span class="citation">[<a href="#ref-Eisenhofer2019">6</a>]</span>. Ordination was carried out with <a href="https://cran.r-project.org/web/packages/Rtsne/Rtsne.pdf">Rtsne</a> (perplexity=90, max_iter=2,000) on a matrix of Spearman’s distances created with the <a href="https://rdrr.io/rforge/ClassDiscovery/">ClassDiscovery</a> package.</p>
<p>Metagenomic assembly was carried out on non-human reads pooled by cancer type with <a href="https://github.com/voutcn/megahit">MEGAHIT</a><span class="citation">[<a href="#ref-RN269">7</a>]</span>. Taxonomic classifications of contigs were obtained with <a href="https://github.com/bbuchfink/diamond">DIAMOND</a><span class="citation">[<a href="#ref-RN204">8</a>]</span> with <a href="https://www.ncbi.nlm.nih.gov/refseq/about/nonredundantproteins/">NCBI non-redundant proteins</a>.</p>
<p>Functional potential of putative proteins was estimated using <a href="https://github.com/tseemann/prokka">Prokka</a><span class="citation">[<a href="#ref-Seemann2014">9</a>]</span> and <a href="https://github.com/ebi-pf-team/interproscan">InterProScan</a><span class="citation">[<a href="#ref-Jones2014">10</a>]</span>.</p>
</div>
<div id="sepath-results" class="section level1">
<h1>SEPATH Results</h1>
<p>Colorectal and oral cancers demonstrate the greatest median number of microbial reads. A background number of classified reads exists throughout all cancer types.</p>
<div class="figure" style="text-align: center"><span id="fig:cancertype"></span>
<img src="images/cancertypesummary.png" alt="Microbial reads in each tumour type" width="100%" />
<p class="caption">
Figure 1: Microbial reads in each tumour type
</p>
</div>
<div id="colorectal-and-oral-cancers-show-distinctive-microbial-communities" class="section level4">
<h4>Colorectal and Oral Cancers Show Distinctive Microbial Communities</h4>
<div class="figure" style="text-align: center"><span id="fig:cancertypetsne"></span>
<img src="images/krakentsne.png" alt="t-SNE plot of cancer samples using Spearman's distance coloured by tumour type. Colorectal and oral cancer are shown in green and blue respectively and separate out into clusters in the bottom half of the plot. This plot was produced using a reduced set of 652/1534 genera." width="108%" />
<p class="caption">
Figure 2: t-SNE plot of cancer samples using Spearman’s distance coloured by tumour type. Colorectal and oral cancer are shown in green and blue respectively and separate out into clusters in the bottom half of the plot. This plot was produced using a reduced set of 652/1534 genera.
</p>
</div>
<p><br></p>
</div>
</div>
<div id="assembly-results" class="section level1">
<h1>Assembly Results</h1>
<p>Assembling microbial reads within each tumour type has resulted in a total of 17.8 million contigs. The number of contigs produced by each cancer type was positively correlated with the number of reads submitted to each assembly (Spearman’s <span class="math inline">\(\rho = 0.87\)</span>)</p>
<div class="figure" style="text-align: center"><span id="fig:asscontaminants"></span>
<img src="Poster_files/figure-html/asscontaminants-1.png" alt="The number of contigs in each assembly after removing mammalian and common contaminant genera. Colorectal samples were excluded due to being prohibitively large to assemble as one pool" width="100%" />
<p class="caption">
Figure 3: The number of contigs in each assembly after removing mammalian and common contaminant genera. Colorectal samples were excluded due to being prohibitively large to assemble as one pool
</p>
</div>
<p><br></p>
</div>
<div id="functional-results" class="section level1">
<h1>Functional Results</h1>
<p>5,264 different pathways were reported across all ontologies, representing ~10% of all known metabolic pathways. This data has been made available via: <a href="https://UEA-Cancer-Genetics-Lab.github.io/Pancancer_Microbial_Pathways/">https://UEA-Cancer-Genetics-Lab.github.io/Pancancer_Microbial_Pathways/</a>.</p>
<p>This has suggested some tantalising pathways for future research such as <em>“PD-L1 expression and PD-1 checkpoint pathway in cancer”</em></p>
<p>It is hoped that this resource can provide researchers with an additional strand of evidence for a non-human pathway existing in cancer.</p>
<p>The number of pathway hits was correlated with the number of assembled contigs (Spearman’s <span class="math inline">\(\rho = 0.92\)</span>) and is therefore sensitive to the sample size. For this reason it is not advisable to investigate differences between cancer types.</p>
<div class="figure" style="text-align: center"><span id="fig:allpathways"></span>
<img src="Poster_files/figure-html/allpathways-1.png" alt="The distribution of pathway hits within all cancer types across all ontologies. The number of pathways for each ontology is demonstrated on the x-axis" width="100%" />
<p class="caption">
Figure 4: The distribution of pathway hits within all cancer types across all ontologies. The number of pathways for each ontology is demonstrated on the x-axis
</p>
</div>
</div>
<div id="conclusions" class="section level1">
<h1>Conclusions</h1>
<ul>
<li><p>SEPATH suggests limited pancancer microbial structure.</p></li>
<li><p>This may be caused by inbalances in reference genomes available.</p></li>
<li><p>Metagenomic assembly is a reference-independent approach that may reveal more about pancancer microbial structure.</p></li>
<li><p>All metabolic pathways reported have been made available for researchers.</p></li>
<li><p>This resource should be used for hypothesis generation or as preliminary evidence for a pathway in cancer.</p></li>
</ul>
</div>
<div id="ongoing-tasks" class="section level1">
<h1>Ongoing Tasks</h1>
<p>The technical difficulties of assembling colorectal data has been circumvented by dividing the pooled reads into six sets. In order to fairly compare cancer types, a single database must be created. To do this, we have concatenated all assembled contigs which contains &gt;18 million contigs. To reduce this database and remove redundancy we have selected representative sequences by clustering with <a href="https://github.com/weizhongli/cdhit/wiki">CD-HIT</a><span class="citation">[<a href="#ref-Li2006">11</a>]</span> which has removed 8 million contigs. Each sample is currently being pseudo-aligned to this pancancer database with <a href="http://pachterlab.github.io/kallisto/">Kallisto</a><span class="citation">[<a href="#ref-Bray2016">12</a>]</span>.</p>
</div>
<div id="acknowledgements" class="section level1">
<h1>Acknowledgements</h1>
<p>This poster was created in <a href="https://github.com/brentthorne/posterdown">posterdown</a> and the code to do so is available on <a href="https://github.com/UEA-Cancer-Genetics-Lab/EACR_Bioinformatics_2021">GitHub</a></p>
<p>Thanks to <a href="http://www.genomicsengland.co.uk">Genomics England</a> including participants and staff as well as <a href="http://www.big-c.co.uk/">Big C</a> and <a href="https://prostatecanceruk.org/">Prostate Cancer UK</a> for supporting this project.</p>
<p><img src="resources/logos.png" width="100%" style="display: block; margin: auto;" /></p>
<div id="references" class="section level3 unnumbered">
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-RN102">
<p>1. Plummer M, Martel C de, Vignat J, Ferlay J, Bray F, Franceschi S. Global burden of cancers attributable to infections in 2012: A synthetic analysis. The Lancet Global Health. 2016;4:e609–16.</p>
</div>
<div id="ref-RN138">
<p>2. Hibbitts S, Tristram A, Beer H, McRea J, Rose B, Hauke A, et al. UK population based study to predict impact of hpv vaccination. J Clin Virol. 2014;59:109–14. doi:<a href="https://doi.org/10.1016/j.jcv.2013.12.002">10.1016/j.jcv.2013.12.002</a>.</p>
</div>
<div id="ref-RN455">
<p>3. Magiorkinis G, Matthews PC, Wallace SE, Jeffery K, Dunbar K, Tedder R, et al. Potential for diagnosis of infectious disease from the 100,000 genomes project metagenomic dataset: Recommendations for reporting results. Wellcome Open Research. 2019;4.</p>
</div>
<div id="ref-RN454">
<p>4. Gihawi A, Rallapalli G, Hurst R, Cooper CS, Leggett RM, Brewer DS. SEPATH: Benchmarking the search for pathogens in human tissue whole genome sequence data leads to template pipelines. Genome Biol. 2019;20:208. doi:<a href="https://doi.org/10.1186/s13059-019-1819-8">10.1186/s13059-019-1819-8</a>.</p>
</div>
<div id="ref-RN72">
<p>5. Wood D, Salzberg S. Kraken - ultrafast metagenomic sequence classification using exact alignments. Genome Biol. 2014;15.</p>
</div>
<div id="ref-Eisenhofer2019">
<p>6. Eisenhofer R, Minich JJ, Marotz C, Cooper A, Knight R, Weyrich LS. Contamination in low microbial biomass microbiome studies: Issues and recommendations. Trends in Microbiology. 2019;27:105–17. doi:<a href="https://doi.org/10.1016/j.tim.2018.11.003">10.1016/j.tim.2018.11.003</a>.</p>
</div>
<div id="ref-RN269">
<p>7. Li D, Liu CM, Luo R, Sadakane K, Lam TW. MEGAHIT: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de bruijn graph. Bioinformatics. 2015;31:1674–6. doi:<a href="https://doi.org/10.1093/bioinformatics/btv033">10.1093/bioinformatics/btv033</a>.</p>
</div>
<div id="ref-RN204">
<p>8. Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using diamond. Nature Methods. 2015;12:59–60.</p>
</div>
<div id="ref-Seemann2014">
<p>9. Seemann T. Prokka: Rapid prokaryotic genome annotation. Bioinformatics. 2014;30:2068–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btu153">10.1093/bioinformatics/btu153</a>.</p>
</div>
<div id="ref-Jones2014">
<p>10. Jones P, Binns D, Chang H-Y, Fraser M, Li W, McAnulla C, et al. InterProScan 5: Genome-scale protein function classification. Bioinformatics. 2014;30:1236–40. doi:<a href="https://doi.org/10.1093/bioinformatics/btu031">10.1093/bioinformatics/btu031</a>.</p>
</div>
<div id="ref-Li2006">
<p>11. Li W, Godzik A. Cd-hit: A fast program for clustering and comparing large sets of protein or nucleotide sequences. Bioinformatics. 2006;22:1658–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btl158">10.1093/bioinformatics/btl158</a>.</p>
</div>
<div id="ref-Bray2016">
<p>12. Bray NL, Pimentel H, Melsted P, Pachter L. Near-optimal probabilistic RNA-seq quantification. Nature Biotechnology. 2016;34:525–7. doi:<a href="https://doi.org/10.1038/nbt.3519">10.1038/nbt.3519</a>.</p>
</div>
</div>
</div>
</div>

</div>
<div class="main">
<p><strong>Microbial DNA in Cancer Sequence Data</strong></p>
</div>
<div class="main_bottom">
<img id="main-img-left" src=>
<img id="main-img-center" src=>
<img id="main-img-right" src=>
</div>
</div>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
var script = document.createElement("script");
script.type = "text/javascript";
var src = "true";
if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
if (location.protocol !== "file:" && /^https?:/.test(src))
src = src.replace(/^https?:/, '');
script.src = src;
document.getElementsByTagName("head")[0].appendChild(script);
})();
</script>


</body>
</html>
